test-butler-app/src/main/aidl/com/linkedin/android/testbutler/ButlerApi.aidl

```diff
@@ -17,6 +17,7 @@ package com.linkedin.android.testbutler;
 
 interface ButlerApi {
     boolean setWifiState(boolean enabled);
+    boolean setGsmState(boolean enabled);
```

Comment:
New methods always need to be added to the **_end_** of the aidl file, [never the beginning or middle](http://stackoverflow.com/questions/20327910/strange-behavior-when-i-add-new-method-in-aidl-file-in-the-beginning). This is so things will still work if someone is using an older version of the library with a newer version of the app.